api_specs/v3/component_analyses_v3.yaml

```diff
@@ -0,0 +1,413 @@
+              description: User uuid.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  anyOf:
+                    - $ref: '#/components/schemas/FreeUserBatch'
+                    - $ref: '#/components/schemas/NoVulnerabilities'
+              examples:
+                FreeUser Batch:
+                  $ref: '#/components/examples/free-user-batch'
+                No Vulnerability Batch:
+                  $ref: '#/components/examples/no-vul-batch'
+
+        202:
+          description: If any of Package is unknown, We will have 202 response status code.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  anyOf:
+                    - $ref: '#/components/schemas/UnknownPkgResponse'
+                    - $ref: '#/components/schemas/FreeUserBatch'
+              examples:
+                All Packages Unknown:
+                  $ref: '#/components/examples/all-packages-unknown'
+                Some Packages Unknown:
+                  $ref: '#/components/examples/some-packages-unknown'
+
+        401:
+          description: Request unauthorized
+          content: {}
+        404:
+          description: Endpoint not found
+          content: {}
+
+
+  /component-vulnerability-analyses/{eco}/{name}/{ver}:
+    get:
+      tags:
+      - CA Singular Call
+      summary: Get vulnerability details about a software component and recommendations
+        to change to a more secured version
+      operationId: da_scanner.api_v3.component-analyses
+      parameters:
+      - name: eco
+        in: path
+        description: component ecosystem
+        required: true
+        schema:
+          type: string
+      - name: name
+        in: path
+        description: component name
+        required: true
+        schema:
+          type: string
+      - name: ver
+        in: path
+        description: component version
+        required: true
+        schema:
+          type: string
+      - name: uuid
+        in: header
+        description: User Identification uuid
+        required: false
+        schema:
+          type: string
+
+      responses:
+        200:
+          description: Detailed response for the requested application component
+          headers:
+            uuid:
+              schema:
+                type: string
+              description: User uuid.
+          content:
+            application/json:
+              schema:
+                items:
+                  oneOf:
+                    - $ref: '#/components/schemas/FreeUser'
+                    - $ref: '#/components/schemas/NoVulnerabilities'
+              examples:
+                FreeUser:
+                  $ref: '#/components/examples/free-user'
+                No Vulnerability:
+                  $ref: '#/components/examples/no-vul'
+
+        202:
```

Comment:
This should change to no vulnerability found for the package.

Reply:
ACK.